cli/messages.go

```diff
@@ -125,17 +125,20 @@ Example: nordvpn set meshnet on`
 	MsgSetMeshnetSuccess            = "Meshnet is set to '%s' successfully."
 	MsgMeshnetAlreadyEnabled        = "Meshnet is already enabled."
 	MsgMeshnetAlreadyDisabled       = "Meshnet is already disabled."
-	MsgMeshnetNotEnabled            = "Meshnet is not enabled."
+	MsgMeshnetNotEnabled            = "Meshnet is not enabled. To enable Meshnet, use command \"nordvpn set meshnet on\"."
 	MsgMeshnetNordlynxMustBeEnabled = "NordLynx technology must be set to use this feature."
-	MsgMeshnetVersionNotSupported   = "Current application version does not support the meshnet feature."
-	MsgMeshnetUsage                 = "Manages mesh network and access to it. In order to enable the feature, execute `nordvpn set meshnet on`"
-	MsgMeshnetRefreshUsage          = "Refreshes the meshnet in case it was not updated automatically."
-	MsgMeshnetPeerUnknown           = "Peer '%s' is unknown."
+	MsgMeshnetVersionNotSupported   = "Current application version does not support the Meshnet feature."
+	MsgMeshnetUsage                 = "Safely access other devices, no matter where in the world they are"
```

Comment:
Pending writer review (although this one is simply copied from our website). 

Also, I asked writer review only for `MsgMeshnetUsage` and `MsgMeshnetNotEnabled` messages as other descriptions are either links or text copied from those links. If you disagree please let me know.